From rdunlap@xenotime.net Sun Oct 23 12:47:06 2005
Date: Sun, 23 Oct 2005 11:58:19 -0700
From: "Randy.Dunlap" <rdunlap@xenotime.net>
Cc: gregkh <greg@kroah.com>
Subject: [PATCH] kernel-doc: fix PCI hotplug
Message-Id: <20051023115819.2755f13d.rdunlap@xenotime.net>

From: Randy Dunlap <rdunlap@xenotime.net>

PCI hotplug.c: does not contain kernel-doc, so don't process it for now.

Signed-off-by: Randy Dunlap <rdunlap@xenotime.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 Documentation/DocBook/kernel-api.tmpl |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- gregkh-2.6.orig/Documentation/DocBook/kernel-api.tmpl
+++ gregkh-2.6/Documentation/DocBook/kernel-api.tmpl
@@ -286,7 +286,9 @@ X!Edrivers/pci/search.c
  -->
 !Edrivers/pci/msi.c
 !Edrivers/pci/bus.c
-!Edrivers/pci/hotplug.c
+<!-- FIXME: Removed for now since no structured comments in source
+X!Edrivers/pci/hotplug.c
+-->
 !Edrivers/pci/probe.c
 !Edrivers/pci/rom.c
      </sect1>
